eosclubhouse/utils.py

```diff
@@ -390,17 +390,32 @@ def get_flatpak_sandbox():
     return sandbox
 
 
+def get_variant_signature(variant):
+    if isinstance(variant, bool):
+        return 'b'
+
+    if isinstance(variant, str):
+        return 's'
+
+    if isinstance(variant, tuple):
+        signature = '('
+        for v in variant:
+            signature += get_variant_signature(v)
+        signature += ')'
+        return signature
+
+    if isinstance(variant, list):
+        signature = 'a' + get_variant_signature(variant[0])
+        return signature
+
+    raise TypeError('Error: value is not a basic type')
+
+
```

Comment:
Can you explain why are these changes needed? If they were only for eosmetrics, I'd left them out of the PR.

Reply:
Yes, you're right, this is not needed, so we can remove these changes from the PR.